sources/packages/backend/apps/queue-consumers/src/processors/assessment/cancel-application-assessment.processor.ts

```diff
@@ -125,6 +129,9 @@ export class CancelApplicationAssessmentProcessor {
         .update(assessment.id, {
```

Comment:
Not a part of the PR and optional, audit fields missing here.

Reply:
I will add since it is important to have it captured 😉

Reply:
Audit fields added.